### ISSUE TITLE
Comments replace method calls

### ISSUE THREAD

**Issue by mdemin914** [dangmai/prettier-plugin-apex#19]
Hey,

This: 
```
public class TestClass {
  static testMethod void test1() {
    PageReference returnPage = someClass.doSomething();
    // check values
    someClass.doSomethingElse();
  }
}
```

Turns into: 
```
public class TestClass {
  static testMethod void test1() {
    PageReference returnPage = someClass.// check values
    doSomething();
    someClass.doSomethingElse();
  }
}
```
Using:
"prettier": "^1.16.4",
"prettier-plugin-apex": "^1.0.0-alpha.7"

**Comment by dangmai** [comment]
Thank you for the bug report! That definitely doesn't look right, I'll try to fix it for the next alpha release. 

Right now comments are still bring worked on very actively - they're surprisingly difficult to get right, so I appreciate all edge case reports like this.

**Comment by dangmai** [comment]
This has been fixed in `1.0.0-alpha.8`. Please upgrade and let me know if it works for you.

Thanks again for the bug report!
